crates/tree_sitter_crochet/corpus/destructuring.txt

```diff
@@ -2,48 +2,24 @@
 Object destructuring assignments
 ================================================================================
 
-({a, b: c.d, ...e[f]} = object);
```

Comment:
It's kind of hard to reason about what's going on here.  I'd like to reserve destructuring for introducing new bindings.  If people need to do what's show here they can do:
```
let {a, b, ...rest} = object;
c.d = b;
e[f] = rest;
```